tests/rkt_caps_test.go

```diff
@@ -87,11 +87,13 @@ func TestCaps(t *testing.T) {
 		patchTestACI(stage2FileName, stage2Args...)
 		defer os.Remove(stage1FileName)
 		defer os.Remove(stage2FileName)
+		ctx := newRktRunCtx()
+		defer ctx.cleanup()
 
 		for _, stage := range []int{1, 2} {
 			t.Logf("Running test #%v: %v [stage %v]", i, tt.testName, stage)
 
-			cmd := fmt.Sprintf("../bin/rkt --debug --insecure-skip-verify run --set-env=CAPABILITY=%d ./rkt-inspect-print-caps-stage%d.aci", int(tt.capa), stage)
+			cmd := fmt.Sprintf("%s --debug --insecure-skip-verify run --set-env=CAPABILITY=%d ./rkt-inspect-print-caps-stage%d.aci", ctx.cmd(), int(tt.capa), stage)
```

Comment:
There is another `../bin/rkt` in this file that could be converted to the new pattern. `git grep ../bin/rkt`? ;)